docs/docs/getting-started/componentDefinition.mdx

```diff
@@ -0,0 +1,13 @@
+A component is a Helm Chart that can be installed on an Airy Cluster in order to extend its functionality.
+
+Components can be characterized by the functionality they enable:
+
+1. UI Components that provide new user interfaces in Airy
+2. Connector Components that connect one or more third-party services with Airy to exchange data. 
+3. API Components that expose additional endpoints to Airy.
+
```

Comment:
The are also `integration` components. Such as the `integration-webhook`.

Reply:
Same as a comment below. We will defer.